frontend/src/app/patients/AddPatient.tsx

```diff
@@ -11,7 +11,10 @@ const AddPatient = () => {
     return <div>"No facility selected"</div>;
   }
   return (
-    <PatientForm patient={{ id: "" }} activeFacilityId={activeFacilityId} />
+    <PatientForm
+      patient={{ patientId: "" }}
```

Comment:
Why was the change from `id` to `patientId` made?

Reply:
the record in `patientForm` [expected `patientId`](https://github.com/CDCgov/prime-simplereport/blob/5eae9c954c5b3e2d4e4f9fad56e5b23f48f3b539/frontend/src/app/patients/PatientForm.tsx#L185). The old js code worked since `undefined` and `""` are both falsy but TS was complaining here.